packages/terra-data-grid/src/DataGrid.module.scss

```diff
@@ -1,25 +1,11 @@
-// Themes
-@import './orion-fusion-theme/DataGrid.module';
-@import './clinical-lowlight-theme/DataGrid.module';
-
 :local {
   .data-grid-container {
     height: 100%;
     overflow: auto;
     width: 100%;
-  }
-
-  .data-grid {
-    // Style for the grid itself
-    border-collapse: separate;
-    border-spacing: 0;
-    table-layout: fixed;
-    user-select: none;
-    width: 1px;
 
-    :focus {
-      outline: var(--terra-data-grid-cell-focus-outline, 3px dashed #000);
-      outline-offset: var(--terra-data-grid-cell-focus-outline-offset, -2px);
```

Comment:
This focus selector should not have been removed.

Reply:
Restored this.

Reply:
It appears that this is still removed.

Reply:
So it turns out that this is in table, so we don't actually need this here after all.